### PR TITLE
--with-default-names not supported when installing grep with Homebrew 2.x

### PR DESCRIPTION
**Pull request by antlong**
Homebrew 2.x dropped support for installing grep --with-default-names, so the argument has to be removed on platforms running v2. I left it for users running 1.x since there wasn't a need to change it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
 ## Install and start with latest Selenium 2
 
-    curl -sSL https://raw.githubusercontent.com/dosel/t/i/p | bash -s 2 start
+    curl -sSL https://raw.githubusercontent.com/antlong/t/i/p | bash -s 2 start
 
 ## Install and start a specific version
 
-    curl -sSL https://raw.githubusercontent.com/dosel/t/i/p | bash -s 3.0.1a start
+    curl -sSL https://raw.githubusercontent.com/antlong/t/i/p | bash -s 3.0.1a start
```
Comment (elgalu) [inline README.md]: `antlong` will pull the script from your fork

:-1:

**Review by elgalu — CHANGES_REQUESTED** [review]

**Review by diemol — CHANGES_REQUESTED** [review]
Hi @antlong, thanks for the PR!
Please just have a look to the comments and then we'll merge it right away.

**Comment by antlong** [comment]
Thanks, let me know if you need anything.

**Review by diemol — APPROVED** [review]
@antlong I just made the changes myself, the rest looks great, thank you for your contribution!

**Pull request merged**